app/models/vacancy.rb

```diff
@@ -126,4 +127,10 @@ def on_expired_vacancy_feedback_submitted_update_stats_updated_at
 
     self.stats_updated_at = Time.current
   end
+
+  def enable_job_applications_cannot_be_changed_once_listed
+    return unless persisted? && listed? && enable_job_applications_changed?
```

Comment:
Nice rhyme

Reply:
Now I'm thinking whether there is a word for "change" that rhymes with -isted 🤔

Reply:
twisted!